HHS/Head-Start-TTADP
# [TTAHUB-551] Create Script to Update Missing Duration Data

## Description of change

Created a script to process a CSV and update duration on AR's.

## How to test

Create a CSV with two column's ID and DUR. Run the script importActivityReportDuration  with your CSV. The durations should be updated on the AR's listed in the CSV.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-551


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
